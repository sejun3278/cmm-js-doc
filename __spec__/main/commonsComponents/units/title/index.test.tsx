import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

import _Title from "src/main/commonsComponents/units/title";
import { useRouter } from "next/router";

// 가짜 router 만들기
jest.mock("next/router", () => ({
  useRouter: jest.fn(),
}));
const push = jest.fn();
(useRouter as jest.Mock).mockImplementation(() => ({
  push,
}));

export default it("Title Units Page Sanpshot", () => {
  const component = render(
    <_Title
      title="타이틀 테스트"
      titleLevel="h2"
      className="_jest_title_"
      styles={{ color: "blue" }}
    />
  );
  // @ts-ignore
  expect(component.container).toMatchSnapshot();
});
