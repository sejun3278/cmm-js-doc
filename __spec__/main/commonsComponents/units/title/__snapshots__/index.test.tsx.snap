// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Units Page Sanpshot 1`] = `
<div>
  <div>
    <h2
      class="_title_ _jest_title_"
      style=""
    />
  </div>
</div>
`;
